blablas/ep196/index.md

```diff
@@ -0,0 +1,62 @@
+---
+date: 2024-11-03
+duration: "2:27:56"
+title: "The illusion of productivity and danger of burnout"
+tags: ["burnout", "psychology"]
+category: "career"
+youtube: https://www.youtube.com/watch?v=3JsNCcz1Hf0
+published: true
+featured: false
+---
+
+This episode focuses on the modern obsession with productivity and its associated risks, such as burnout and declining mental health. The discussion covers practical solutions to advance in one’s career without compromising mental well-being.
+
+## Guests
+
+- [Achraf Aamri](https://linkedin.com/in/achrafaamri)
+- [Adnan Merrakchi](https://twitter.com/adnanmrakchi)
+
+## Notes
+
+00:00:00 - Lmo9adimma
+
+00:04:35 - Achnu huwa burnout bnsba l our guests?
+
+00:10:10 - Personal Experiences and The problem of self-indentification
+
+00:25:45 - Adnane on Work life balance and Discipline 
+
+00:28:39 - Office vs Remote and effortlessness vs Will-power
+
+00:30:15 - Advice to remote workers
+
+00:36:20 - On romanticizing the Job and discipline
+
+00:39:20 - How delegation fights burnout
+
+00:44:40 - The illusion of productivity and Respecting constraints
+
+01:04:20 - Knowing yourself your contsraints and avoiding comparisons
+
+01:17:07 - Why not push to the limit? and achraf's story
+
+01:32:20 - Mourad on the measure of success
+
+01:35:20 - Burnout VS imposter syndrome
+
+01:47:00 - How do guests reduce burnout effects?
+
+02:11:20 - Guests sharing their favorite books
+
+02:16:20 - Information flux and burnout
+
+02:19:20 - Featuring Comments
+
+## Links
+
+
```

Comment:
No links ?

Reply:
Just to confirm, the script didn't manage to extract links or there is no links in the first place

Reply:
There were no links in the episode in the first place